### DISCUSSION TITLE
I want to know the reliable installation of Georgia-ReBORN theme.

### DISCUSSION THREAD

**Discussion by eco-pimento** [TT-ReBORN/Georgia-ReBORN#60] [Q&A] [answered]
A sad event happened to TT-ReBORN. I managed to operate TT-ReBORN-Georgia-ReBORN, but I mistakenly clicked Default instead of Georgia-ReBORN with View button ⇒ Layout button ⇒ 3 choices, and the Georgia-ReBORN theme became Default. No matter what I did, I couldn't go back.

I wish I had exported the Georgia-ReBORN theme as an FCL file, but I tried again from the beginning, but it didn't work. Even if I follow the instructions, it doesn't work. I have a portable installation, can you tell me how to succeed without the FCL files?

Certainly, I have a memory that I could set in the fth file, but I did it properly and I do not remember clearly. Please help me.

**Comment by TT-ReBORN** [comment, marked as answer]
Hey, I'm not dead yet! I guess you can blame your translator :p

Go to your foobar/profile folder and delete the configuration folder. 
Now from the [zip](https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip) extract the configuration folder and paste it in your foobar/profile folder again. 
Start foobar and select Columns UI and all should be good...

-TT

**Reply by eco-pimento** [reply]
TT, thank you for letting me know the solution by email immediately. I am using it without any problems. I'm glad that the Diskart, logo, and music folders are also included. Updated the foo_spider_monkey_panel.js file. I backed up skn with an fcl file and theme with an fth file.
I also copied and backed up the foober2000 folder.
I will use it as a companion for a long time. Thank you so much.

**Reply by TT-ReBORN** [reply]
You don't need any fcl or fth file, only the foo_ui_columns.dll.cfg that is located in your profile/configuration folder.

-TT

**Comment by eco-pimento** [comment]
TT, thank you for letting me know the solution by email immediately. I am using it without any problems. I'm glad that the Diskart, logo, and music folders are also included. Updated the foo_spider_monkey_panel.js file. I backed up skn with an fcl file and theme with an fth file.

I also copied and backed up the foober2000 folder.

I will use it as a companion for a long time. Thank you very much.
